setup.py

```diff
@@ -44,7 +44,7 @@ def run_tests(self):
     package_data={'stripe': ['data/ca-certificates.crt']},
     zip_safe=False,
     install_requires=[
-        'requests >= 2',
+        'requests >= 2; python_version > "3.0"',
```

Comment:
Super minor nit, but could you change the condition to use `>=` instead of `>`?

We don't support Python 3.0 so this doesn't really change anything, but it bothers me that this implies that if 3.0 were supported, then `requests` would not be necessary.

Reply:
Done